content/courses/native-onchain-development/paging-ordering-filtering-data-frontend.md

```diff
@@ -162,21 +171,23 @@ For example, you could search through a list of contacts by including a `memcmp`
 filter:
 
 ```tsx
+const DATA_OFFSET = 2; // The offset into the program account data - Skip the first 2 bytes of metadata
+const DATA_LENGTH = 18; // The length of the data slice to retrieve - Retrieve 18 bytes of relevant data starting from the 3rd byte
```

Comment:
Why do we specifically skip the first 2 bytes of metadata?
Why do we retrieve specifically 18 bytes of relevant data?

 (Don't answer, just put the reason into the code or comment)